### ISSUE TITLE
Update the Training, Validation and Holdout Set Stats

### ISSUE THREAD

**Issue by mjoshua97241** [mjoshua97241/bank-fraud-detection-project#10]
Change the following to 
- [ ] - Training Set (70%) -> 55%
- [ ] - Validation Set (15%) -> 15%
- [ ] - Holdout Set (15%) -> 30%
